data/json/npcs/robofac/robofac_intercom/robofac_intercom_prototypes.json

```diff
@@ -261,7 +261,7 @@
         "text": "How are my armor repairs coming?",
         "condition": {
           "and": [
-            { "math": [ "time_since(u_timer_hub_rnd_u_waiting_on_hub)", "<", "time('1 d')" ] },
+            { "math": [ "time_since(u_timer_hub_rnd_u_waiting_on_hub)", ">", "time('1 d')" ] },
```

Comment:
This is for "it's not ready yet" so the sign is correct

Reply:
Does the second commit correct? I assume the idea is to have separate timers for weapon and for armor, but didn't dig deep enough